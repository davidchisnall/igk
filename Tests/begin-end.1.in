\begin{itemize}
	\item{Item 1}
		\begin{item}
			\enumerate{
			\item{Item 2a}\item{Item 2b}
			}
		\end{item}
\end{itemize}
