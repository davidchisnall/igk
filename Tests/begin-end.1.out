\itemize{
	\item{Item 1}
		\item{
			\enumerate{
			\item{Item 2a}\item{Item 2b}
			}
		}
}
